giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/model/impl/File.java

```diff
@@ -48,6 +48,9 @@ public class File implements IFile {
     private ProcessingStatus processingStatus;
     private String groupId;
     private String recordId;
+    /**
+    * This is the list of old file version IDs for the file if the file was reprocessed.
+    */
```

Comment:
this doesn't really explain what it is for. The javadoc should answer the questions:
- what is an "old file version"?
- why do they exist when a file was reprocessed?
- why do we store those ids?